product_docs/docs/epas/16/reference/oracle_compatibility_reference/epas_compat_cat_views/pg_views/pg_namespace.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "PG_NAMESPACE"
+---
+
+The `PG_NAMESPACE` view describes namespaces, which are the structures underlying SQL schemas. Each namespace can have a separate collection of relations, types, and so forth without name conflicts. In addition to the information in the [PostgreSQL pg_namespace description](https://www.postgresql.org/docs/current/catalog-pg-namespace.html), EDB Postgres Advanced Server provides the following information:
+
+| Name               | Type    | Description                          |
+| ------------------ | ------- | ------------------------------------ |
+| nspbodysource      | text    |                                      |
```

Comment:
i don't see this column in EPAS or PG.

Reply:
These are the columns which are additionally added in EPAS 

nspparent          | oid       |           | not null | 
 nspobjecttype      | oid       |           | not null | 
 nspforeignserver   | oid       |           | not null | 
 nspsecdef          | boolean   |           | not null | 
 nspcompoundtrigger | boolean   |           | not null | 
 nspremoteschema    | text      | C         |          | 
 nspheadsrc         | text      | C         |          | 
 nspbodysrc